pages/1.12/cli/command-reference/dcos-security/dcos-security-cluster/dcos-security-cluster-secret-store-unseal/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: layout.pug
+navigationTitle:  dcos security cluster secret-store unseal
+title: dcos security cluster secret-store unseal
+menuWeight: 115
+excerpt: Unsealing a secret store
+enterprise: true
+---
+# Description
+
+The `dcos security cluster secret-store unseal` command allows you to unseal a given store.
+
+# Usage
+
+```
+dcos security cluster secret-store unseal [OPTIONS] STORE_ID KEY
+```
+
+# Options
+
+| Name |  Description |
+|---------|-------------|
+| `STORE_ID KEY`  | ID key of secret store. (Required) |
```

Comment:
These are two distinct parameters. They should be placed on separate lines. 

@mhrabovcin What are good descriptions for these parameters?

Reply:
Per your comment above, I will list this as a Positional Argument.